### PR TITLE
Shared buffer

### PR DESCRIPTION
**Pull request by kjatkows**
--disable_blitter -> disabled blitter on input and output of kernels

### CODE REVIEW

File: include/ze_info/zenon.hpp [hunk 1]
```diff
@@ -61,6 +61,9 @@ class zenon
     bool log;
     bool multi_ccs = true;
     void* input1_buffer = nullptr, * input2_buffer = nullptr, * output_buffer = nullptr, * im_buf1 = nullptr, * im_buf2 = nullptr, * im_buf3 = nullptr, * im_buf4 = nullptr, * im_buf5 = nullptr, * im_buf6 = nullptr;
+    void* sharedBuffer = nullptr;
+    ze_host_mem_alloc_desc_t hostDesc = { ZE_STRUCTURE_TYPE_HOST_MEM_ALLOC_DESC };
+
```
Comment (tponieck) [inline include/ze_info/zenon.hpp:66]: remove empty line

File: include/ze_info/zenon.hpp [hunk 2]
```diff
@@ -113,6 +116,8 @@ class zenon
     ze_kernel_timestamp_result_t kernel_ts_results[MAX_EVENTS_COUNT];
     uint32_t graph_event_count = 0;
     std::vector<std::string> kernel_names;
+
+
```
Comment (tponieck) [inline include/ze_info/zenon.hpp]: remove empty lines

File: src/zenon.cpp [hunk 3]
```diff
@@ -269,6 +269,9 @@ void zenon::allocate_buffers()
     SUCCESS_OR_TERMINATE( zeMemAllocDevice( context, &memory_descriptor,
         alloc_size, 1, device, &im_buf6 ) );
 
+    hostDesc.flags = ZE_HOST_MEM_ALLOC_FLAG_BIAS_UNCACHED;
+    SUCCESS_OR_TERMINATE(zeMemAllocShared(context, &memory_descriptor, &hostDesc, alloc_size, 1, device, &sharedBuffer));
```
Comment (tponieck) [inline src/zenon.cpp]: shouldn't this be done only if user wants no blitter mode?

Comment (kjatkows) [reply src/zenon.cpp]: done

**Pull request merged**